extension-style/src/main/java/com/mapbox/maps/extension/style/utils/TypeUtils.kt

```diff
@@ -297,31 +296,19 @@ inline fun <reified T> StylePropertyValue.silentUnwrap(): T? {
  * Extension function for [Feature] to convert [Feature] to [Value].
  */
 fun Feature.toValue(): Value {
-  val expected = ValueConverter.fromJson(this.toJson())
-  expected.value?.let {
-    return it
-  }
-  throw RuntimeException(expected.error)
+  return TypeUtils.wrapToValue(this.toJson())
 }
 
 /**
  * Extension function for [FeatureCollection] to convert [FeatureCollection] to [Value].
  */
 fun FeatureCollection.toValue(): Value {
-  val expected = ValueConverter.fromJson(this.toJson())
-  expected.value?.let {
-    return it
-  }
-  throw RuntimeException(expected.error)
+  return TypeUtils.wrapToValue(this.toJson())
```

Comment:
@tobrun I think it would be better to move this temporary optimization to `GeoJsonSource`'s feature, featureCollection, geometry methods. The `toValue` methods could be useful in some cases.

Reply:
TypeUtils is an internal class and adopted functions are only used by GeoJsonSource